features/command_line/bisect.feature

```diff
@@ -155,3 +155,30 @@ Feature: Bisect
       """
     When I run `rspec ./spec/calculator_10_spec.rb[1:1] ./spec/calculator_1_spec.rb[1:1] --seed 1234`
     Then the output should contain "2 examples, 1 failure"
+
+  Scenario: Use `--bisect` flag in `SPEC_OPTS` to create a minimal repro case for the ordering dependency
+    When I run `rspec --seed 1234`
+    Then the output should contain "10 examples, 1 failure"
+    When I set the environment variables to:
+      | variable  | value    |
+      | SPEC_OPTS | --bisect |
+    And I run `rspec --seed 1234`
+    Then bisect should succeed with output like:
+      """
+      Bisect started using options: "--seed 1234"
+      Running suite to find failures... (0.16755 seconds)
+      Starting bisect with 1 failing example and 9 non-failing examples.
+      Checking that failure(s) are order-dependent... failure appears to be order-dependent
+
+      Round 1: bisecting over non-failing examples 1-9 .. ignoring examples 6-9 (0.30166 seconds)
+      Round 2: bisecting over non-failing examples 1-5 .. ignoring examples 4-5 (0.30306 seconds)
+      Round 3: bisecting over non-failing examples 1-3 .. ignoring example 3 (0.33292 seconds)
+      Round 4: bisecting over non-failing examples 1-2 . ignoring example 1 (0.16476 seconds)
+      Bisect complete! Reduced necessary non-failing examples from 9 to 1 in 1.26 seconds.
+
+      The minimal reproduction command is:
+        rspec ./spec/calculator_10_spec.rb[1:1] ./spec/calculator_1_spec.rb[1:1] --seed 1234
+      """
+    When I run `rspec ./spec/calculator_10_spec.rb[1:1] ./spec/calculator_1_spec.rb[1:1] --seed 1234`
+    Then the output should contain "2 examples, 1 failure"
+
```

Comment:
I think so, yes - I lobbed it in to get something to work against, but without my fix this test will forkbomb any machine it runs on, so I don't think it should stick around. 😄

Now that it's working, I can update the assertions made about the `Bisect::Runner` commands to check that `--bisect` is dropped from the env passed to `Open3` - I think that should be sufficient.